<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translatorman</class>
 <widget class="QMainWindow" name="Translatorman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Translatorman</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mui_tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>25</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionReport_bug"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Translatorman"/>
    <addaction name="actionAbout_Pamcli"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_all"/>
    <addaction name="actionCloseall_except"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="action_Select_all"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_Replace"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionZoom_in"/>
     <addaction name="actionZoom_out"/>
     <addaction name="actionReset"/>
    </widget>
    <addaction name="actionFull_screen"/>
    <addaction name="actionShow_toolbar"/>
    <addaction name="actionShow_graphic"/>
    <addaction name="actionShow_in_ngroff"/>
    <addaction name="separator"/>
    <addaction name="menuZoom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="menuToolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="windowTitle">
    <string>Listados</string>
   </property>
   <property name="toolTip">
    <string>Toolbar</string>
   </property>
   <property name="statusTip">
    <string>Toolbar</string>
   </property>
   <property name="whatsThis">
    <string>Toolbar</string>
   </property>
   <property name="accessibleName">
    <string>Toolbar</string>
   </property>
   <property name="accessibleDescription">
    <string>Toolbar</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Images/document_properties.svg</normaloff>:/Images/document_properties.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFull_screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Full screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAbout_Translatorman">
   <property name="text">
    <string>&amp;About Translatorman</string>
   </property>
   <property name="toolTip">
    <string>About Translatorman</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="iconText">
    <string>Paises / Provincias</string>
   </property>
   <property name="toolTip">
    <string>Paises y provincias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>&amp;Find...</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="text">
    <string>&amp;Replace...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Images/document_save.svg</normaloff>:/Images/document_save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Show_splited">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show splited</string>
   </property>
  </action>
  <action name="actionShow_graphic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;graphic</string>
   </property>
  </action>
  <action name="actionShow_in_ngroff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show in &amp;ngroff</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionAbout_Pamcli">
   <property name="text">
    <string>About &amp;Pamcli</string>
   </property>
  </action>
  <action name="actionReport_bug">
   <property name="text">
    <string>Report &amp;bug</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Pref&amp;erences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Images/document_open.svg</normaloff>:/Images/document_open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Images/edit_delete.svg</normaloff>:/Images/edit_delete.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>&amp;Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="action_Select_all">
   <property name="text">
    <string>&amp;Select all</string>
   </property>
  </action>
  <action name="actionCloseall_except">
   <property name="text">
    <string>Close others</string>
   </property>
  </action>
  <action name="actionShow_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;toolbar</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
